lib/kube/proxy/resource_list.go

```diff
@@ -71,6 +75,49 @@ func (f *Forwarder) listResources(sess *clusterSession, w http.ResponseWriter, r
 		status = rw.Status()
 	} else {
 		allowedResources, deniedResources := sess.Checker.GetKubeResources(sess.kubeCluster)
+
+		details, err := f.findKubeDetailsByClusterName(sess.kubeClusterName)
+		if err != nil {
+			return nil, trace.Wrap(err, "failed to find kube details by cluster name %q", sess.kubeClusterName)
+		}
+		makeKind := func(resources []types.KubernetesResource) []types.KubernetesResource {
+			var out []types.KubernetesResource // Not pre-allocating as we don't know the final size. Likely 0 if no CRD resource defined.
+		loop:
+			for _, r := range resources {
+				tmp := strings.Split(r.Kind, "/")
+				if len(tmp) < 3 { // Expect <group>/<version>/<kind>. If not, skip.
+					continue
```

Comment:
Something should be logged here to make finding mistakes easier, unless there is some validation elsewhere that alerts the user to invalid config.